plugins/filter_aws/aws.c

```diff
@@ -1048,6 +1116,14 @@ static struct flb_config_map config_map[] = {
      "if both tags_include and tags_exclude are specified, configuration is invalid"
      " and plugin fails"
     },
+    {
+     FLB_CONFIG_MAP_INT, "retry_interval_s", "300",
+     /* for the time being, the only group which has retries is ec2 tags */
+     /* therefore configuration immediately sets this value just for ec2 tags */
+     0, FLB_TRUE, offsetof(struct flb_filter_aws,
+             group_tag.retry_required_interval),
+     "Defines minimum duration between retries for fetching EC2 instance tags"
```

Comment:
I thought we discussed, that if we add a generic named retry setting, it needs to be for all metadata. 

Otherwise its confusing and then also if we want to change it to support all groups in the future when we add specific retry settings, it would be a backwards breaking change.

Reply:
Okay. I appear to have misinterpreted something, my apologies.

I prefer not to introduce new configuration parameters that would be deprecated quite soon. Therefore, I will implement retries for all groups in this pull request. Let me know if that's okay for you.

Reply:
Cool!